airbyte-webapp/src/index.tsx

```diff
@@ -7,7 +7,7 @@ import { loadDatadog } from "utils/datadog";
 import { loadOsano } from "utils/dataPrivacy";
 import { loadSentry } from "utils/sentry";
 
-import "./globals";
+import "./dayjs-setup";
```

Comment:
:information_source: This is required, due to some missmatch in `require` paths, we can't have this file named be `globals` in `src` directly :shrug: No one knows exactly why the pathes are missmatched, but this is the easiest workaround for now.